barretenberg/sol/src/honk/optimised/generate_offsets.py

```diff
@@ -3,9 +3,9 @@
 ## A mini python script to help generate the locations in memory of the indicies requred to generate a proof
 
 vk_fr = [
```

Comment:
I think this script needs to be updated (for INVERTED_CHALLENEGE_POW_MINUS_U_27_LOC etc)